buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseGenerator.kt

```diff
@@ -31,25 +31,69 @@ import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
 import org.gradle.kotlin.dsl.findByType
 
+data class ReleaseReport(
```

Comment:
I think it's fine to not provide documentation for *everything* added here- but if you could provide some for `ReleaseReport` it would help a lot in unfamiliar folk coming across this (or even future you and I).

Reply:
renamed for clarity and added some documentation